pyproject.toml

```diff
@@ -34,7 +34,7 @@ optional = true
 
 [tool.poetry.group.dev.dependencies]
 pytest = "^6.2.4"
-pyright = "^0.0.12"
+pyright = "^1.1.291"
```

Comment:
Why are we asking for an older minimum version than we want to run with? We are locked on the latest one and have to deal with the type errors from the latest one anyway.

Reply:
Updated to match the current version.

Or did you propose to not lock in the project file at all?